marine_benchmark.c

```diff
@@ -148,13 +147,13 @@ int main(int argc, char *argv[]) {
     };
 
     benchmark_case cases[] = {
```

Comment:
Maybe add benchmark cases with another encapsulation to see how it affects memory and speed?
Ideally you should run all of the cases with ethernet and then all of the cases with another encapsulation type.

However, I believe this is optional and up to @yehonatanz to decide :)

Reply:
I agree, it would be the best but definitely optional